pkg/security/secl/rules/model.go

```diff
@@ -149,6 +149,7 @@ type SetDefinition struct {
 	Size         int                    `yaml:"size" json:"size,omitempty"`
 	TTL          *HumanReadableDuration `yaml:"ttl" json:"ttl,omitempty"`
 	Private      bool                   `yaml:"private" json:"private,omitempty"`
+	Inherited    bool                   `yaml:"inherited" json:"inherited,omitempty"`
```

Comment:
Can we make it so this new field is reported in ruleset_loaded events?

Reply:
Done